sklearn/metrics/scorer.py

```diff
@@ -44,7 +45,54 @@
 from ..base import is_regressor
 
 
-class _BaseScorer(metaclass=ABCMeta):
+class _MultimetricScorer(dict):
+    """Callable dictionary for multimetric scoring."""
```

Comment:
Please briefly describe keys being strings and values being instances of `_BaseScorer`

I first thought (without looking) that this was also a `_BaseScorer` instance

Reply:
Would have been nice, the passthrough scorer and custom scorers may have weird interfaces, so `_MultimetricScorer.__call__` needed to be as generic as possible.